projects/media-viewer/src/public_api.ts

```diff
@@ -7,3 +7,4 @@ export * from './lib/toolbar/toolbar.module';
 export * from './lib/comments-summary/comments-summary.module';
 export * from './lib/annotations/annotation-api.service';
 export * from './lib/annotations/annotations.module';
+export * from './lib/viewers/error-message/viewer-exception.model';
```

Comment:
I'm sure this works, but is it better to have it inside the media-viewer.module file?

Reply:
as that gets exported anyway? like we have done for toolbar module